src/services/twitter.service.ts

```diff
@@ -0,0 +1,118 @@
+import Environment from "../environment";
+import axios, { AxiosRequestConfig } from 'axios';
+
+export type Tweet = {
```

Comment:
y not interface?

Reply:
I doesn't make any difference in this instance to use a `type` over an `interface`, but I can change it if this is part of the style.